src/mongo_client.ts

```diff
@@ -318,36 +318,15 @@ const kOptions = Symbol('options');
  * The programmatically provided options take precedence over the URI options.
  *
  * @example
- * ```js
- * // Connect using a MongoClient instance
- * const MongoClient = require('mongodb').MongoClient;
- * const test = require('assert');
- * // Connection url
- * const url = 'mongodb://localhost:27017';
- * // Database Name
- * const dbName = 'test';
- * // Connect using MongoClient
- * const mongoClient = new MongoClient(url);
- * mongoClient.connect(function(err, client) {
- *   const db = client.db(dbName);
- *   client.close();
- * });
- * ```
+ * ```ts
+ * import { MongoClient } from 'mongodb';
```

Comment:
Same question as the doc comment on db.  What is the purpose of the comment, and why are we only demonstrating command monitoring but otherwise not mentioning it at all?

Reply:
to give an example of what this thing is for, it's the first thing you see when you land on the API page for `MongoClient`, we have extensive docs on how to use command monitoring on the manual site, this is just to kick off what area of the API you're looking at.